automatic/qtox/qtox.nuspec

```diff
@@ -27,10 +27,6 @@ qTox is a powerful [Tox](https://tox.chat/) client that follows the Tox design g
 * Emoticons
 * Auto-updates on Windows and Mac, packages on Linux
 * And many more options!
-
-### Package notes
-
-This package will always install the latest version of qTox, regardless of the package version.
     </description>
     <projectUrl>https://tox.chat/clients.html#qtox</projectUrl>
```

Comment:
Not releated to your changes but projectUrl is also outdated. Should be https://qtox.github.io/